src/form-behavior/like.js

```diff
@@ -0,0 +1,31 @@
+import symbols from "../util/symbols.js";
+import { resolveValue } from "../util.js";
+import { getRole } from "./role.js";
+
+const { formBehavior, internals } = symbols.known;
+
+export const hooks = {
+	first_connected () {
+		let { like, role, valueProp = "value", changeEvent = "input" } = this.constructor[formBehavior];
```

Comment:
```suggestion
		if (!this.constructor[formBehavior]) {
			return;
		}

		let { like, role, valueProp = "value", changeEvent = "input" } = this.constructor[formBehavior];
```

We should do the stuff only with suitable instances.